packages/switch-button/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 Oscar Braunert <o@ovl.design>
```

Comment:
I need to update this. Maybe «YYYY tournant <team@tournnant.dev>»

I would suggest that I do it in Communard and replace existing notes once I installed the new version here.

Reply:
👍